lib/lookup.json

```diff
@@ -503,6 +503,8 @@
   },
   "torrent-stream": {
     "prefix": "v",
+    "flaky": ["ppc", "rhel"],
```

Comment:
Is this flaky on all rhel platforms (e.g. rhel8-x64, rhel8-arm64, rhel8-ppc64le, rhel8-s390x)?

Reply:
thanks for making me double check @richardlau !

now that I looked at results again it seems to be failing in many more platforms 😞 

- [win-vs2019](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3298/nodes=win-vs2019/testReport)
- [rhel8-ppc64le](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3297/nodes=rhel8-ppc64le/testReport)
- [ubuntu1804-64](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3300/nodes=ubuntu1804-64/testReport)
- [rhel8-x64](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3300/nodes=rhel8-x64/testReport)
- [rhel8-s390x](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3300/nodes=rhel8-s390x/testReport)
- [aix72-ppc64](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3300/nodes=aix72-ppc64/testReport)
- [rhel8-ppc64le](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3300/nodes=rhel8-ppc64le/testReport)